apps/editing-toolkit/editing-toolkit-plugin/coming-soon/coming-soon.php

```diff
@@ -20,7 +20,7 @@ function should_show_coming_soon_page() {
 	$share_code = get_share_code();
 	if ( is_accessed_by_valid_share_link( $share_code ) ) {
 		track_preview_link_event();
-		setcookie( 'wp_share_code', $share_code, time() + 3600, '/', false, is_ssl() );
+		setcookie( 'wp_share_code', $share_code, time() + 3600, '/', false, is_ssl(), true );
```

Comment:
Also fixed in the PR moving this code to Jetpack repo: https://github.com/Automattic/jetpack/pull/27815/files#diff-fdd367b51a6fa0f82a7ec1faa3c7751756f98acd665f37618bde58e7766adb00R23